client/components/SingleProduct.js

```diff
@@ -18,7 +22,8 @@ class SingleProduct extends Component {
   }
 
   render() {
-    const product = this.props.singleProduct
+    console.log(this.props)
```

Comment:
niiiice

Reply:
yaa because we're using a keyed object (instead of an array) in our redux store, have to wrap it back up to use it like normal.

on second thought tho its probably better practice to wrap it up in our render function instead of mapState.
that way we can still get that sick 0(1)